transport_nantes/mobilito/views.py

```diff
@@ -436,3 +452,50 @@ def get(self, request: HttpRequest, *args, **kwargs) -> HttpResponse:
         context = self.get_context_data(**kwargs)
         self.check_view_permission(context["mobilito_session"])
         return self.render_to_response(context)
+
+
+@csrf_protect
+def report_session(request: HttpRequest, **kwargs) -> HttpResponse:
+    """Report a session as inappropriate"""
+
+    if request.method == 'POST':
+        if request.user.is_authenticated:
+            user = request.user
+        else:
+            user = None
+
+        session_sha1 = kwargs['session_sha1']
+        try:
+            # In the event that somehow, in spite of CSRF protection, a user
```

Comment:
The goal of CSRF is not to avoid flagging non-existent sessions, it is to avoid outsiders doing it without going through our website.

Reply:
I agree. I hardly see how this could happen to be honest.